MOBILE_API.xml

```diff
@@ -3293,6 +3293,9 @@
         <param name="scale" type="Float" minvalue="1" maxvalue="10" mandatory="false" since="6.0">
             <description>The scaling factor the app should use to change the size of the projecting view.</description>
         </param>
+        <param name="preferredFPS" type="Integer" minvalue="0" maxvalue="2147483647" mandatory="false">
```

Comment:
```suggestion
        <param name="preferredFPS" type="Integer" minvalue="0" maxvalue="2147483647" mandatory="false" since="7.1">
```
You will also need to merge develop into this branch for this to work

Reply:
Thanks for the review, @jacobkeeler 
I have updated MOBILE_API as you suggested.